src/test/java/org/fundacionjala/pivotal/cucumber/steps/ProjectSteps.java

```diff
@@ -3,6 +3,7 @@
 import cucumber.api.java.en.And;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this unused import 'cucumber.api.java.en.And'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AUselessImportCheck)